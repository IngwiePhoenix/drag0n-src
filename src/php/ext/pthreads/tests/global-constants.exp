string:string(8) "mystring"
long:int(10)
double:float(1.19)
null:NULL
boolean:bool(true)