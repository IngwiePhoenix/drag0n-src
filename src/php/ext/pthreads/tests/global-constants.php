<?php
define ("SCONST", "mystring");
define ("LCONST", 10);
define ("DCONST", 1.19);
define ("NCONST", null);
define ("BCONST", true);

class TestThread extends Thread {
	public function run() {
		foreach (array(
			"string" => SCONST,
			"long" => LCONST,
			"double" => DCONST,
			"null" => NCONST,
			"boolean" => BCONST
		) as $key => $constant) {
			printf("%s:", $key);
			var_dump($constant);
		}
	}
}

$thread = new TestThread();
$thread->start();
?>
