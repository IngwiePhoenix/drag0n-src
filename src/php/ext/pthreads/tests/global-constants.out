Warning: Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) in Unknown on line 0
string:string(8) "mystring"
long:int(10)
double:float(1.19)
null:NULL
boolean:bool(true)